src/components/Team/teamCard.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import Image from 'next/image';
+import AdityaKulkarni from '../../public/Team/Executives/Aditya Kulkarni.jpg';
+import AnaqiAmir from '../../public/Team/Executives/Anaqi Amir.jpg';
+import ChristineVan from '../../public/Team/Executives/Christine Van.png';
+import HibahMehvish from '../../public/Team/Executives/Hibah Mehvish.jpg';
+import KhadeemSankar from '../../public/Team/Executives/Khadeem Sankar.jpg';
+import MalharPandya from '../../public/Team/Executives/Malhar Pandya.jpg';
+import Jean from '../../public/Team/Executives/Jean.jpg';
+import TiannePane from '../../public/Team/Executives/Tianne Pane.jpg';
+
+const profileImages: Record<string, any> = {
+  'Aditya Kulkarni': AdityaKulkarni,
+  'Anaqi Amir': AnaqiAmir,
+  'Christine Van': ChristineVan,
+  'Hibah Mehvish': HibahMehvish,
+  'Khadeem Sankar': KhadeemSankar,
+  'Malhar Pandya': MalharPandya,
+  Jean: Jean,
+  'Tianne Pane': TiannePane,
+};
```

Comment:
Move the above Record into a separate file, good to keep the Card Component as clean as possible

Use this a guide for a team-config.ts inplace for the `team.json`
```
import { type StaticImageData } from 'next/image';

import Aditya_Kulkarni from '@/public/Team/Executives/Aditya Kulkarni.jpg';
... Image Imports

export interface TeamMember {
  name: string;
  position: string;
  src: StaticImageData;
  alt: string;
}

type TeamGroups = 'Executives';

export const team: Record<TeamGroups, TeamMember[]> = {
  Executives: [
    {
      name: 'Aditya Kulkarni',
      position: 'President',
      src: Aditya_Kulkarni,
      alt: 'Aditya Kulkarni',
    },
  ]
};

```